deepchecks/vision/vision_data.py

```diff
@@ -362,6 +362,10 @@ def __len__(self):
         """Return the number of batches in the dataset dataloader."""
         return len(self._data_loader)
 
+    def assert_image_formatter_valid(self):
+        if not self._has_images:
+            raise DeepchecksValueError('Can\'t run since batch_to_images() was not implemented correctly')
```

Comment:
Explain here how to run the validation that will then tell you what was not done correctly.